mne/io/proj.py

```diff
@@ -669,11 +669,6 @@ def make_eeg_average_ref_proj(info, activate=True, verbose=None):
     eeg_proj: instance of Projection
         The SSP/PCA projector.
     """
-    if info.get('custom_ref_applied', False):
-        raise RuntimeError('Cannot add an average EEG reference projection '
-                           'since a custom reference has been applied to the '
-                           'data earlier.')
-
```

Comment:
and you need to set `info['custom_ref_applied'] = False` in this conditional right?